packages/clay-icon/src/index.tsx

```diff
@@ -26,6 +27,12 @@ const ClayIcon = React.forwardRef<SVGSVGElement, IProps>(
 	({className, spritemap, symbol, ...otherProps}: IProps, ref) => {
 		let spriteMapVal = React.useContext(ClayIconSpriteContext);
 
+		React.useEffect(() => {
+			svg4everybody({
+				polyfill: true,
+			});
+		}, []);
+
```

Comment:
@kresimir-coko I was thinking that we would add this only on clayui.com, because we already have it in DXP, so maybe add it to the doc template.

Reply:
Good idea, it didn't occur to me, but it works! 👍